### PR TITLE
PAR // @VivianSolide

### PR DESCRIPTION
**Pull request by VivianSolide**

### CODE REVIEW

File: starter-code/js/stack.js [hunk 1]
```diff
@@ -3,15 +3,59 @@ var stack = new StackDataStructure();
...
+    }
+    else if (counter === 8){
+      $('body > section > div.books.col-lg-9 > div:nth-child(3)').toggleClass('active');
+      $('body > section > div.books.col-lg-9 > div:nth-child(3) > p').text(stack.stackControl[7]);
+    }
+    else if (counter === 9){
+      $('body > section > div.books.col-lg-9 > div:nth-child(2)').toggleClass('active');
+      $('body > section > div.books.col-lg-9 > div:nth-child(2) > p').text(stack.stackControl[8]);
+    }
+    else if (counter === 10){
```
Comment (ta-web-paris) [inline starter-code/js/stack.js]: waouh that's a lot of "if" ! I am sure that a for loop can help.

File: starter-code/js/stack.js [hunk 2]
```diff
@@ -3,15 +3,59 @@ var stack = new StackDataStructure();
...
+    counter--;
+  }
 });
-//// toggle
-//$('body > section > div.books.col-lg-9 > div:nth-child(10)').toggleClass('active');
-//$('body > section > div.books.col-lg-9 > div:nth-child(10) > p').text(stack.stackControl[0]);
\ No newline at end of file
+//// remove
+$('button.add').click(function () {
+  /// j'ai le poids de ouf...
```
Comment (ta-web-paris) [inline starter-code/js/stack.js]: Keuah ?

Comment (VivianSolide) [reply starter-code/js/stack.js]: C'était hier très tard...

File: starter-code/js/stack.js (hunk 1 above)
Comment (VivianSolide) [reply starter-code/js/stack.js]: I tried while loop but the result wansn't efficient

**Comment by ta-web-paris** [comment]
Un peu bourrin mais ça passe

**Pull request closed without merging**